dwitter/dweet/views.py

```diff
@@ -13,6 +14,7 @@ def handler404(request):
 
 
 @xframe_options_exempt
+@cache_page(3600)
```

Comment:
As there's currently no way to change dweets themselves, this cache shouldn't be a problem for now. Just have to make sure to remember this in the future.